cnf-tests/Dockerfile.openshift

```diff
@@ -12,8 +12,10 @@ WORKDIR $PKG_PATH
 RUN make test-bin
 
 FROM openshift/origin-base
+RUN mkdir -p /usr/local/etc/cnf
```

Comment:
should we put these changes also in the `cnf-tests/Dockerfile` file?

Reply:
Done